src/ga.js

```diff
@@ -175,11 +182,16 @@ function getCpmDistribution(cpm) {
   return distribution;
 }
 
+exports.prefixedSend = function(){
```

Comment:
Does that `exports.prefixedSend` need to be inside the `enableAnalytics` function?